data-sets-api-server/src/main/java/org/zowe/data/sets/controller/DataSetsController.java

```diff
@@ -5,7 +5,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2016, 2019
+ * Copyright IBM Corporation 2016, 2019, 2020
```

Comment:
Copyright is supposed to bookend, so this should be Copyright IBM Corporation 2016, 2020

Reply:
thanks again, just modified